services/ui-src/src/utils/api/requestMethods/report.test.ts

```diff
@@ -0,0 +1,13 @@
+import { getReport, writeReport } from "./report";
+// utils
+import { mockReportData } from "utils/testing/setupJest";
+
+describe("Test banner methods", () => {
+  test("getReport", () => {
+    expect(getReport("ZA3141")).toBeTruthy();
```

Comment:
How is this working without mocking anything?

Reply:
in`services/ui-src/src/utils/testing/setupJest.tsx` we have a global mock on all API calls. I'm not sure how else to test this method because all it does is construct an object with the input and make an api call.

```
jest.mock("aws-amplify", () => ({
  ...
  API: {
    get: () => {},
    post: () => {},
    del: () => {},
    configure: () => {},
  },
}));
```

Reply:
Maybe a better test would be to setup a spy and check that it:
* calls the get()
* with expected expected params